pkg/controller/bastion/actuator.go

```diff
@@ -213,3 +217,19 @@ func deleteRule(openstackClientFactory openstackclient.Factory, ruleID string) e
 	}
 	return client.DeleteRule(ruleID)
 }
+
+func bastionConfigCheck(bastionConfig *controllerconfig.BastionComputeConfig) error {
+	config := controllerconfig.BastionComputeConfig{}
+	if *bastionConfig == config {
+		return errors.New("bastionConfig must not be empty in 00-componentconfig.yaml")
+	}
```

Comment:
instead, check if nil and return an error. Drop the `00-componentconfig.yaml` mention from the error message

Reply:
committed in fc634219